metadata/algolia/meta/system-objecttype-extensions.xml

```diff
@@ -163,6 +163,26 @@ Setting this preference replaces the first two segments, the final index name be
                 <min-length>0</min-length>
             </attribute-definition>
 
+            <attribute-definition attribute-id="Algolia_RecordModel">
+                <display-name xml:lang="x-default">Record model</display-name>
+                <description xml:lang="x-default">Define the model used to store products in Algolia. 'Variation Product': create one record per variation product.
+                    'Base Product': create one record per base product, containing all variants in a 'variants' attribute."</description>
+                <type>enum-of-string</type>
+                <mandatory-flag>false</mandatory-flag>
+                <externally-managed-flag>false</externally-managed-flag>
+                <value-definitions>
```

Comment:
```suggestion
                <value-definitions default="true">
```

Reply:
`[2024-01-31 09:46:48.919 GMT] ERROR [174|51] cvc-complex-type.3.2.2: Attribute 'default' is not allowed to appear in element 'value-definitions'.`

Reply:
ahh sorry, it is my bad :)